installer/installer.py

```diff
@@ -994,6 +1068,29 @@ def ask_for_cosmovisor(self):
             self.is_cosmo_needed = False
         else:
             failure_exit(f"Invalid input provided during installation.")
+    
+    def ask_for_cosmovisor_bump(self):
+        self.exec("echo 'export DAEMON_NAME=cheqd-noded' >> ~/.bashrc")
+        self.exec(f"echo 'export DAEMON_HOME={self.home_dir}/.cheqdnode' >> ~/.bashrc")
+        self.exec(f"echo 'export DAEMON_DATA_BACKUP_DIR={self.home_dir}/.cheqdnode' >> ~/.bashrc")
+        std_out = """{}""".format(self.exec("cosmovisor version 2> /dev/null || echo false").stdout)     
+        arr = std_out.split()
+        current_version = None
+        for i in range (0, len(arr)):
+            if arr[i].lower() == 'version:':
+                # set current version as well as replace if v prefix there
+                current_version = arr[i+1].split("\\")[0].replace('v','')
+                break
+        # if current version is lower than latest ask for upgrade
```

Comment:
1. Have an is_cosmovisor exists, where you check is it installed, where is it installed, and what version it is
2. If it's installed and older version, ask for bump